plugins/services/src/js/components/ServiceBreadcrumbs.js

```diff
@@ -230,7 +249,13 @@ class ServiceBreadcrumbs extends React.Component {
         }
 
         return (
-          <Breadcrumb key={index} title={ids.slice(0, index + 1).join("/")}>
+          <Breadcrumb
+            key={index}
+            title={ids.slice(0, index + 1).join("/")}
+            instancesCount={instancesCount}
```

Comment:
I don't think these service-specific props need to be given anymore.

Reply:
Good catch.